vulkano/src/pipeline/graphics/mod.rs

```diff
@@ -2572,6 +2573,130 @@ impl GraphicsPipelineCreateInfo {
+                } && (!dynamic_state.contains(&DynamicState::PrimitiveTopology)
+                    || match device
+                        .physical_device()
+                        .properties()
+                        .dynamic_primitive_topology_unrestricted
+                    {
+                        Some(b) => !b,
+                        None => false,
+                    })
+                {
+                    return Err(Box::new(ValidationError {
+                        problem: "`input_assembly_state.topology` is not compatible with the \
+                            conservative rasterization mode"
+                            .into(),
+                        vuids: &["VUID-VkGraphicsPipelineCreateInfo-conservativePointAndLineRasterization-08892"],
+                        ..Default::default()
+                    }));
+                }
+            }
+
+            if let (Some(geometry_stage), Some(_), Some(conservative_rasterization_state)) = (
+                geometry_stage,
+                input_assembly_state,
+                conservative_rasterization_state,
+            ) {
+                let spirv = geometry_stage.entry_point.module().spirv();
+                let entry_point_function = spirv.function(geometry_stage.entry_point.id());
+
+                let mut invalid_output = false;
+
+                for instruction in entry_point_function.execution_modes() {
+                    if let Instruction::ExecutionMode { mode, .. } = *instruction {
+                        match mode {
+                            ExecutionMode::OutputPoints => {
+                                invalid_output = true;
+                                break;
+                            }
+                            ExecutionMode::OutputLineStrip => {
+                                invalid_output = true;
+                                break;
+                            }
+                            _ => {}
+                        }
+                    }
+                }
+
+                if match conservative_rasterization_state.mode {
+                    ConservativeRasterizationMode::Disabled => false,
+                    _ => true,
+                } && invalid_output
+                {
+                    return Err(Box::new(ValidationError {
+                        problem: "the output topology of the geometry shader is not compatible with the \
+                            conservative rasterization mode"
+                            .into(),
+                        vuids: &["VUID-VkGraphicsPipelineCreateInfo-conservativePointAndLineRasterization-06760"],
+                        ..Default::default()
+                    }));
+                }
+            }
+
+            if let (Some(mesh_stage), Some(_), Some(conservative_rasterization_state)) = (
+                mesh_stage,
+                input_assembly_state,
+                conservative_rasterization_state,
+            ) {
```

Comment:
```suggestion
            if let (Some(mesh_stage), Some(conservative_rasterization_state)) = (
                mesh_stage,
                conservative_rasterization_state,
            ) {
```

Pipelines with mesh shaders have no input assembly state.